tools/windows-node-installer/pkg/cloudprovider/aws/ec2.go

```diff
@@ -197,14 +198,34 @@ func (a *AwsProvider) CreateWindowsVM() (credentials *types.Credentials, err err
 	// Build new credentials structure to be used by other actors. The actor is responsible for checking if
 	// the credentials are being generated properly. This method won't guarantee the existence of credentials
 	// if the VM is spun up
-	credentials = types.NewCredentials(instanceID, publicIPAddress, decryptedPassword, winUser)
+	credentials := types.NewCredentials(instanceID, publicIPAddress, decryptedPassword, winUser)
+	w.Credentials = credentials
+
+	// Setup Winrm and SSH client so that we can interact with the Windows Object we created
+	// TODO: Add a struct called Connectivity which has information related to Winrm, SSH and have
+	//		getters and setters for it so that it can be exposed a public method
+	// 		jira: https://issues.redhat.com/browse/WINC-239
```

Comment:
Thanks for adding this. However it would be better if you added this as a comment in the `Windows` struct where these fields are defined. That seems like the best common place to add this comment as the placement here could imply it is ec2 specific.